test/parallel/test-http-allow-req-after-204-res.js

```diff
@@ -23,12 +23,14 @@
 const common = require('../common');
 const http = require('http');
 const assert = require('assert');
+const Countdown = require('../common/countdown');
 
 // first 204 or 304 works, subsequent anything fails
 const codes = [204, 200];
 
 // Methods don't really matter, but we put in something realistic.
 const methods = ['DELETE', 'DELETE'];
```

Comment:
this was basically used as the counter, so i do not think it's necessary anymore (hardcoding the method into the request is enough).

Reply:
@Bamieh : Thanks. I've updated the PR. Kindly review!